lib/active_record/connection_adapters/clickhouse_adapter.rb

```diff
@@ -351,8 +351,8 @@ def create_table(table_name, **options, &block)
         end
       end
 
-      def create_function(name, body)
-        fd = "CREATE FUNCTION #{apply_cluster(quote_table_name(name))} AS #{body}"
+      def create_function(name, body, force: false)
+        fd = "CREATE#{' OR REPLACE' if force} FUNCTION #{apply_cluster(quote_table_name(name))} AS #{body}"
```

Comment:
The other methods accept an options hash instead of kwargs, should probably do the same here.

```suggestion
      def create_function(name, body, **options)
        fd = "CREATE#{' OR REPLACE' if options[:force]} FUNCTION #{apply_cluster(quote_table_name(name))} AS #{body}"
```